cfy_manager/components/components_dependencies.py

```diff
@@ -1,15 +1,18 @@
 DEPENDENCIES_ERROR_MESSAGES = {
     'sudo': 'necessary to run commands with root privileges',
     'openssl-1.0.2k': 'necessary for creating certificates',
+    'openssl-1.1.1k': 'necessary for creating certificates',
     'logrotate': 'used in Cloudify logs',
     'systemd-sysv': 'required to create Cloudify services',
+    'systemd': 'required to create Cloudify services',
```

Comment:
...what? Why? We explicitly don't want this in a supervisord setup and everything should work in that case. What's the failure that led to this being added?